java-desktop/src/main/java/com/couchbase/todo/model/Config.java

```diff
@@ -0,0 +1,103 @@
+package com.couchbase.todo.model;
+
+import java.util.Objects;
+
+import com.couchbase.lite.ConsoleLogger;
+import com.couchbase.lite.Database;
+import com.couchbase.lite.LogDomain;
+import com.couchbase.lite.LogLevel;
+import com.couchbase.todo.TodoApp;
+
+
+public final class Config {
+
+    private final boolean loggingEnabled;
```

Comment:
Very nice.  Love it.
Formatting:  Within a class the order is:
All static members/blocks come first
All instance members/blocks come after
Within one of those groups, the order is
all constants (static final, CAPITALIZED)
all types (inner classes, static or instance)
all variables
functions/methods.